source/robot_model/src/Model.cpp

```diff
@@ -326,8 +318,8 @@ Eigen::VectorXd Model::cwln_repulsive_potential_field(const state_representation
 state_representation::JointPositions
 Model::inverse_kinematics(const state_representation::CartesianPose& cartesian_pose,
                           const state_representation::JointPositions& joint_positions,
-                          const std::string& frame_name,
-                          const InverseKinematicsParameters& parameters) {
+                          const InverseKinematicsParameters& parameters,
+                          const std::string& frame_name) {
   unsigned int frame_id;
```

Comment:
Do we even need this whole logic for the frame_id? It only seems to be used once and we can as well call `forward_kinematics` with the frame_name...

Reply:
I also noticed that when using it for the IV. Yes you can probably make another PR unifying a bit those tests